src/app/content/types.ts

```diff
@@ -46,6 +46,7 @@ export interface PageReference {
 }
 
 export interface BookWithOSWebData extends ArchiveBook {
+  book_state: 'coming soon' | 'deprecated' | 'live' | 'retired';
```

Comment:
It's `coming_soon` as [shown here](https://cms-staging.openstax.org/apps/cms/api/v2/pages/?type=books.Book&limit=250&book_state=coming_soon&fields=book_state)